optaweb-vehicle-routing-frontend/src/components/LocationList.tsx

```diff
@@ -62,32 +68,41 @@ const renderLocationList = ({
   isDemoLoading,
 }: ILocationListProps) => {
   return (
-    <Card>
-      <CardHeader>
-        Distance: {distance}
-        <br />
-        Locations: {locations.length}
-        <TripData
-          maxDistance={maxDistance}
-          distance={parseInt(distance, 10) || maxDistance}
-        />
-        <br />
-        <Button
-          type="button"
-          isDisabled={isDemoLoading}
-          style={{ width: '100%' }}
-          onClick={clearHandler}
-        >
-          Clear
-        </Button>
-        <br />
-      </CardHeader>
+    <Card >
+      { /* TO BE REMOVED? Disabled*/
```

Comment:
Yes, please remove this card completely (together with TripData.tsx). We'll need to make a new design a decide where to show details (number of visits, depots, vehicles, road distance, driving time). Until that time, let's remove it.